devops/windows/run--base-docker-wsl/start.ps1

```diff
@@ -0,0 +1,280 @@
+$script_path = $MyInvocation.MyCommand.Path
```

Comment:
What's the difference between this script and the one at `devops/windows/run--base-docker-linux/start.ps1`?

Reply:
the run--base-docker-docker-linux is for windows running as guest in vbox ---   wsl is not available for this --- so we run the docker stack on the linux host environment --- this scripts runs on the linux host

the run--base--docker-wsl runs the docker stack in a wsl linux console ---- ie all windows